src/gui/qgsadvanceddigitizingdockwidget.h

```diff
@@ -32,8 +32,8 @@ class QgsPoint;
 
 // tolerances for soft constraints (last values, and common angles)
 // for angles, both tolerance in pixels and degrees are used for better performance
-static const double SOFT_CONSTRAINT_TOLERANCE_PIXEL = 15;
-static const double SOFT_CONSTRAINT_TOLERANCE_DEGREES = 10;
+static const double SOFT_CONSTRAINT_TOLERANCE_PIXEL = 15; SIP_SKIP
+static const double SOFT_CONSTRAINT_TOLERANCE_DEGREES = 10; SIP_SKIP
```

Comment:
you have to write SIP_SKIP before the semi-colon.
aslo you have to `#include "qgis.h";` on top of the header to get the macros definitions.